course_project/ansible/ami_db.yml

```diff
@@ -0,0 +1,14 @@
+- name: Setup DB server
+  hosts: all
+  become: yes
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: Ensure apt cache is up-to-date
+      apt:
+        update_cache: yes
+        cache_valid_time: 3600
+
+  roles:
```

Comment:
+1 for not re-inventing the wheel and using self-wrote code along with community roles.
Occam would be pleased )